Gems/ScriptEvents/Code/Source/ScriptEventsAssetRef.cpp

```diff
@@ -120,7 +120,7 @@ namespace ScriptEvents
     void ScriptEventsAssetRef::OnAssetUnloaded(
         [[maybe_unused]] const AZ::Data::AssetId assetId, [[maybe_unused]] const AZ::Data::AssetType assetType)
     {
-        if (ScriptEventsAsset* ebusAsset = m_asset.GetAs<ScriptEventsAsset>())
+        if (/*ScriptEventsAsset* ebusAsset = */m_asset.GetAs<ScriptEventsAsset>())
```

Comment:
This function is now not doing anything.
You could delete the content of it.

Reply:
I intentionally commented it out instead of deleting the variable (and the contents of the function) because I am not the original author, and do not know the intent of commenting this out (vs just removing it)

Reply:
Then let's get the opinion of the original author. @lsemp3d Is there a reason why the commented code for ScriptEventsAssetRef::OnAssetUnloaded is still in the code?

Reply:
Looking at the history, there is no indication that it was ever not-disabled. I will remove it